services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/attachment/taydennys/TaydennysAttachmentServiceITest.kt

```diff
@@ -248,7 +248,7 @@ class TaydennysAttachmentServiceITest(
             val failure = assertFailure {
                 attachmentService.addAttachment(
                     taydennysId = UUID.randomUUID(),
-                    attachmentType = ApplicationAttachmentType.MUU,
+                    attachmentType = MUU,
```

Comment:
Why are you changing these? It has nothing to do with this PR.

I think it's clearer to specify the enum class as part of the type. It makes it clear that this is an attachment type of the value other and not some other other or the sound a cow makes.

Reply:
Put `ApplicationAttachmentType.` back.